smurfs/src/reducers/index.js

```diff
@@ -14,6 +14,45 @@
  }
 */
 
+ function rootReducer(state={ smurfs : []}, action) {
+
+    switch (action.type) {
+      
+      case "LOAD":
```

Comment:
You may want to utilize the DOUBLE_NAME convention in your action types as load, success, add, and banish are not very descriptive of what is occurring and hot also have no error state in the event of unsuccessful api calls, how will you handle an error beyond console.logging it? You're users are mossy likely not going to look to the console if something doesn't work